manager/manager.go

```diff
@@ -1008,7 +1044,12 @@ func (self *manager) watchForNewContainers(quit chan error) error {
 			case event := <-self.eventsChannel:
 				switch {
 				case event.EventType == watcher.ContainerAdd:
-					err = self.createContainer(event.Name, event.WatchSource)
+					switch event.WatchSource {
+					case watcher.Rkt:
```

Comment:
Yeah, I guess this makes sense. Can you add a comment here explaining why this is necessary in the Rkt case?